### PR TITLE
[Anchor-201] Update OpenAPI spec

### PR DESCRIPTION
**Pull request by Ifropc**
<details>
  <summary>PR Checklist</summary>

### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `paymentservice.stellar`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
</details>

### What

* Updating OpenAPI specs
* Matching order of OpenAPI in the code

### Why

To better reflect schema of GET response and better rendering [in the documentation](https://developers-pr101.previews.kube001.services.stellar-ops.com/ap_api/resources/get-transaction) 

### Known limitations

N/A

### CODE REVIEW

**Review by lijamie98 — CHANGES_REQUESTED** [review]
Can we avoid modifying code for this PR?

**Review by JakeUrban — APPROVED** [review]

**Comment by Ifropc** [comment]
@lijamie98 I can submit separate pr for formatting and rebase this one.

File: docs/03 - Implementing the Anchor Server/Communication/Platform API.yml [hunk 1]
```diff
@@ -85,14 +84,9 @@ paths:
...
-                type: object
-                properties:
-                  records:
-                    type: array
-                    items:
-                      $ref: './schemas.yml#/components/schemas/Transaction'
-                  cursor:
-                    type: string
+                oneOf:
+                  - $ref: './schemas.yml#/components/schemas/AllTransactionsSEP24'
```
Comment (lijamie98) [inline docs/03 - Implementing the Anchor Server/Communication/Platform API.yml:88]: How do we return the list of transactions of all seps?

File: docs/03 - Implementing the Anchor Server/Communication/Platform API.yml [hunk 2]
```diff
@@ -124,7 +120,7 @@ paths:
                   records:
                     type: array
                     items:
-                      $ref: './schemas.yml#/components/schemas/Transaction'
+                      $ref: './schemas.yml#/components/schemas/MutableTransaction'
```
Comment (lijamie98) [inline docs/03 - Implementing the Anchor Server/Communication/Platform API.yml:123]: Why do we want to call this `MutableTransaction`?

File: docs/03 - Implementing the Anchor Server/Communication/Platform API.yml (hunk 1 above)
Comment (Ifropc) [reply docs/03 - Implementing the Anchor Server/Communication/Platform API.yml:88]: It's the same endpoint. I just wanted to show that get transaction can return either SEP-31 or SEP-24

**Review by lijamie98 — APPROVED** [review]
LGTM

**Pull request merged**